src/Libraries/Microsoft.Extensions.AI.OpenAI/OpenAIAssistantChatClient.cs

```diff
@@ -284,13 +284,18 @@ void IDisposable.Dispose()
                     switch (tool)
                     {
                         case AIFunction aiFunction:
-                            bool? strict = aiFunction.AdditionalProperties.TryGetValue(nameof(strict), out var strictValue) && strictValue is bool strictBool ?
+                            bool? strict = aiFunction.AdditionalProperties.TryGetValue(OpenAIClientExtensions.StrictKey, out var strictValue) && strictValue is bool strictBool ?
                                 strictBool :
                                 null;
+
+                            JsonElement jsonSchema = strict is true ?
```

Comment:
Consider hiding this logic behind a method in `OpenAIClientExtensions` that accepts `strict` as a parameter.

Reply:
Done.